media-libs/libpng/libpng12-1.2.56.recipe

```diff
@@ -21,8 +21,8 @@ SECONDARY_ARCHITECTURES="x86_gcc2"
 
 PROVIDES="
 	libpng12$secondaryArchSuffix = $portVersion compat >= 1.2
-	lib:libpng$secondaryArchSuffix = 3.54.0 compat >= 3
-	lib:libpng12$secondaryArchSuffix = 3.54.0 compat >= 3
+	lib:libpng$secondaryArchSuffix = 3.56.0 compat >= 3
+	lib:libpng12$secondaryArchSuffix = 0.56.0 compat >= 0
```

Comment:
you shouldn't "fix" things like that: these versions are there since a while and possibly used by other packages. Such a change can break these other packages for no reason, no one will rebuild these packages. Please stay with 3.56.0 compat >= 3 on lib:libpng12 and devel:libpng12